### ISSUE TITLE
Black theme

### ISSUE THREAD

**Issue by xvbcfj** [zed-industries/extensions#1181]
### Check for existing issues

- [X] Completed

### Misc notes

Similar to https://plugins.jetbrains.com/plugin/19348-true-dark or https://marketplace.visualstudio.com/items?itemName=viktorqvarfordt.vscode-pitch-black-theme 

Basically, full black background colors but decent accessibility (so border colors etc should be decently visible).
